tests/integration/helpers/cluster.py

```diff
@@ -1452,6 +1466,8 @@ def create_dir(self, destroy_dir=True):
 
         if self.stay_alive:
             entrypoint_cmd = CLICKHOUSE_STAY_ALIVE_COMMAND
+            
+        entrypoint_cmd = entrypoint_cmd.replace("{main_config_file}", self.main_config_name)
```

Comment:
This replacement logic is repeated many times, and that's not good.
Let's add a new variable `self.clickhouse_start_command`, do this replacement only once in `__init__()`, and then use this variable everywhere instead of `CLICKHOUSE_START_COMMAND` and `CLICKHOUSE_STAY_ALIVE_COMMAND`.